ballerina/Ballerina.toml

```diff
@@ -12,57 +12,72 @@ license = ["Apache-2.0"]
 [build-options]
 observabilityIncluded=true
 
-[platform.java11]
+[platform.java17]
 graalvmCompatible = true
 
-[[platform.java11.dependency]]
-path = "../native/build/libs/azure-cosmosdb-native-4.2.0.jar"
+[[platform.java17.dependency]]
 groupId = "io.ballerinax"
 artifactId = "cosmosdb"
 version = "4.2.0"
+path = "../native/build/libs/azure-cosmosdb-native-4.2.0.jar"
 
-[[platform.java11.dependency]]
+[[platform.java17.dependency]]
+groupId = "com.azure"
+artifactId = "azure-cosmos"
+version = "4.52.0"
+path = "./lib/azure-cosmos-4.52.0.jar"
+
+[[platform.java17.dependency]]
 groupId = "io.netty"
 artifactId = "netty-resolver-dns"
-version = "4.1.94.Final"
+version = "4.1.100.Final"
+path = "./lib/netty-resolver-dns-4.1.100.Final.jar"
```

Comment:
Is it mandatory to provide the path for maven dependencies? IINM it is not

Reply:
If we don't provide the path, each Ballerina build will download the dependencies. This way, we handle the maven dependencies through the Gradle build, which is efficient.

Reply:
didn't know that. it makes sense now :) thanks for the insight